docs/aries-cloud-agent-demo.md

```diff
@@ -90,17 +99,35 @@ At this point, the two ACAs are connected to each other.
 
 ### Dependencies
 
-Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start. 
 
 Install Aries cloud-agents (for more info see <a href="https://github.com/hyperledger/aries-cloudagent-python#install" target=_blank>here</a>) if you do not have it on your machine:
 
 ``` bash
 pip install aries-cloudagent
 ```
 
+This demo has been successfully tested with aca-py version 0.4.5.
+
+This demo requires an instance of von network running in docker locally (for more info see <a href="https://github.com/bcgov/von-network#running-the-network-locally" target=_blank>here</a>)
+
+This demo has been successfully tested with the von-network git repository pulled on 07 Aug 2020. 
```

Comment:
You should point to a specific commit or tag (ideally have the full command to clone the repo in the correct state)

Reply:
Done